tests/posix/common/src/mutex.c

```diff
@@ -213,20 +214,19 @@ static void timespec_add_ms(struct timespec *ts, uint32_t ms)
 static void *test_mutex_timedlock_fn(void *arg)
 {
 	struct timespec time_point;
-	pthread_mutex_t *mutex = (pthread_mutex_t *)arg;
+	pthread_mutex_t *mtx = (pthread_mutex_t *)arg;
```

Comment:
Was the name changed to avoid a compiler warning about a local shadowing a global?

Reply:
That was my assumption